core/src/main/java/org/eclipse/hono/tracing/TracingHelper.java

```diff
@@ -373,31 +372,6 @@ public static SpanContext extractSpanContext(final Tracer tracer, final JsonObje
                 : null;
     }
 
-    /**
-     * Injects a {@code SpanContext} into the headers of vert.x {@code DeliveryOptions}.
-     *
-     * @param tracer The Tracer to use for injecting the context.
-     * @param spanContext The context to inject or {@code null} if no context is available.
-     * @param deliveryOptions The delivery options to inject the context into.
-     * @throws NullPointerException if tracer or deliveryOptions is {@code null}.
-     */
-    public static void injectSpanContext(final Tracer tracer, final SpanContext spanContext, final DeliveryOptions deliveryOptions) {
```

Comment:
FMPOV we could/should keep this method for when we indeed want to pass on a tracing context via the event bus ...

Reply:
ok, I've removed the 2nd commit where methods got removed.